app/views/pdfjs_viewer/viewer/_viewer.html.erb

```diff
@@ -42,6 +42,7 @@ See https://github.com/adobe-type-tools/cmap-resources
       </script>
     <% end %>
     <%= javascript_include_tag "pdfjs_viewer/application" %>
+    <%= javascript_tag "window.HOSTED_VIEWER_ORIGINS = #{PdfjsViewer.hosted_viewer_origins.to_json.html_safe};" %>
```

Comment:
I don't think configuring the viewer in the browser is a good idea. This check is a safeguard to prevent malicious requests. The browser environment is not something to be trusted.